### ISSUE TITLE
Broken link to "Understanding masking and padding" guide

### ISSUE THREAD

**Issue by TheCrazyT** [keras-team/keras-nlp#1446]
**Describe the bug**
The link https://keras.io/guides/understanding_masking_and_padding/ that is referenced at https://github.com/keras-team/keras-nlp/blob/ce93ca8bd6a84121f72ab0ea6bfc63083222f9ad/keras_nlp/layers/modeling/transformer_decoder.py#L41 is not working.

**Expected behavior**
A valid page, currently "404: Page not found" appears.

**Additional context**
-

**Would you like to help us fix it?**
Sadly can not guess the right url ...

**Comment by TheCrazyT** [comment]
maybe it is a bug in the website itself because on git the guide seem to exist: https://github.com/keras-team/keras-io/blob/master/guides/md/understanding_masking_and_padding.md

**Comment by mattdangerw** [comment]
I think this guide got removed during the Keras 3 migration. I will pull it down for now.
